models/Discord/Commands/DiscordCommandNotes.js

```diff
@@ -13,7 +13,7 @@ class DiscordCommandNotes extends DiscordCommand {
       return;
     }
 
-    var ckey = args[0];
+    var ckey = args[0].replaceAll("[^a-zA-Z0-9]", "");;
```

Comment:
Extra colon here.

Reply:
also replaceAll is not a function. This is javascript, not java.

do `.replace(/[^a-z0-9]/gi, "")`